webapp-django/crashstats/crashstats/models.py

```diff
@@ -674,10 +674,13 @@ def kwargs_to_params(self, kwargs):
 
         for param in self.get_annotated_params():
             name = param['name']
-            if param['required'] and not kwargs.get(name):
-                raise RequiredParameterError(name)
             value = kwargs.get(name)
-            if not value:
+
+            # 0 is a perfectly fine value, it should not be considered "falsy".
+            if param['required'] and not value and value != 0:
+                raise RequiredParameterError(name)
+
+            if not value and value != 0:
                 continue
```

Comment:
I can't put my finger on it but it feels strange that you're doing the ` ... != 0` check twice.